### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by bjorobinson**
Pull Request for end of sprint 1

### CODE REVIEW

File: POCPs/app.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+//wait equivalent time for 1 whole beat
+function waitBeat(){
+  spawnRest();
+  spawnRest();
+}
+
+//Base function for the 'waiting' - it waits equivalent time for 1/2 a beat
+function spawnRest(){
+  setTimeout(function(){
+    // console.log("rest");
```
Comment (JSChilds) [inline POCPs/app.js]: Don't leave commented-out code in your final product.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Functions broadly work, but there are no clear instructions, there's no song to play along to and it doesn't really look presentable as a game.

**Comment by bjorobinson** [comment]
Pull Request for sprint 2

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  background-position-x: -712px;
+  background-size: 200%;
+}
+
+/* Class Styling */
+.col-xs-6{
+  margin: none;
+  padding: 0px 0px;
+}
+
```
Comment (JSChilds) [inline css/style.css]: The class above and below are the same, target them both to stop it from being redundant.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      </div>
+    </div>
+    <div id="myModal" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h4 class="modal-title">You Rock</h4>
+          </div>
+          <div id="score_result" class="modal-body">
+            PH Text
```
Comment (JSChilds) [inline index.html]: Put this in a p tag or something, don't have text just in a div.

File: js/app.js [hunk 4]
```diff
@@ -0,0 +1,223 @@
+$(document).ready(function(){
```
Comment (JSChilds) [inline js/app.js:1]: All your code should be indented within this document.ready function.

File: js/app.js [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+  showScore();
+  $("#start_btn").click(function(){
+    playSong();
+    $("#start_btn").hide();
```
Comment (JSChilds) [inline js/app.js]: Semicolons missing from end of lines 6 and 7.

File: README.md [hunk 6]
```diff
@@ -1 +1,143 @@
...
+  var keycode = event.which
+  delete input[keycode];
+  removePushed($("#" + whatButton(keycode)));
+});
+```
+`input` in this program is an object, as opposed to, for example, an array. When a button is pressed, the colour keycode is then created as a key to the object. This had the following benefits:
+* Javascript objects cannot have duplicate keys, so repeated calls of the keydown function (through holding the key down) will do nothing.
+* `delete object[key]` is far easier than removing a member from an array, which may have required me to keep constant track of the index and contents of the input array.
+
+# The Game
```
Comment (JSChilds) [inline README.md]: Put the section about the game first.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Please make requested changes.

**Comment by bjorobinson** [comment]
Requested Changes have been made

**Review by JSChilds — APPROVED** [review]

**Pull request merged**